docs/config.md

```diff
@@ -1333,7 +1333,30 @@ as follows:
 backends.ssh.allowed-signers = "/path/to/allowed-signers"
 ```
 
-### Sign commits only on `jj git push`
+### Manually signing commits
+
+You can use [`jj sign`](./cli-reference.md#jj-sign)/[`jj unsign`](./cli-reference.md#jj-unsign)
+to sign/unsign commits manually.
+
+!!! warning
+
+    `jj sign` always signs commits, even if they are already signed by the
+    user. While this is cumbersome for users signing via hardware devices, we
+    cannot reliably check if a commit is already signed without creating a
+    signature (see [this issue](https://github.com/jj-vcs/jj/issues/5786)).
+
+### Automatically signing commits
+
+The `signing.behavior` configuration option has four different options for what
+to do with signing commits on modification of a change (e.g., rebasing or edits).
```

Comment:
Just wanted to note that this also applies for newly created changes. I tried to write up this behavior in https://github.com/jj-vcs/jj/pull/5935/commits/066355bda4d84c92721a939100cc0156c05bd797, but the wording can probably be improved. Some additional comments: https://github.com/jj-vcs/jj/pull/5935#discussion_r1989811146.